systemtest/tests/integration/test.sh

```diff
@@ -42,7 +42,6 @@ EOF
 ./integration-tests/scripts/run-local-candlepin.sh
 
 # create testing data in local candlepin
-./integration-tests/scripts/post-activation-keys.sh
```

Comment:
I'm not against of removing this line as a part of this PR, but please add some meaningful commit message to related commit.

Reply:
ok next time I will add it.

Reply:
This change should be part of different commit or please remove this change from this PR.